### ISSUE TITLE
Can't set custom class in chart

### ISSUE THREAD

**Issue by jackjoy** [telerik/kendo-angular#1016]
In following use case,we want to use custom class to identify different chart control:

```
<kendo-chart class="chart1"><kendo-chart>
<kendo-chart class="chart2"><kendo-chart>
```
But it not working. I think the cause is the misuse of HostBinding in chart component:
```
@HostBinding('class')
```

which should be:
```
@HostBinding('class.k-widget')
@HostBinding('class.k-chart')
```

**Comment by gyoshev** [comment]
The fix is available on the [dev channel](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/), version `v1.3.2-dev.201710231409`.

**Comment by ipeshev** [comment]
Fixed in v1.3.2-dev.201710231409

**Comment by gyoshev** [comment]
Released officially in `v1.3.2`.
